src/components/view-only.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Button } from "./common-components";
+import { ImportButtonsComponent } from "./schedule-page/import-buttons/import-buttons.component";
+import RouteButtonsComponent from "./common-components/route-buttons/route-buttons.component";
+import ManagementPage from "./workers-page/management-page.component";
+import { ScheduleComponent } from "./schedule-page/table/schedule/schedule.component";
+
+interface TabData {
+  label: string;
+  component: JSX.Element;
+  rightSideButtons: JSX.Element;
+}
+
+export function ViewOnlyComponent(): JSX.Element {
```

Comment:
Tutaj analogicznie nazwałbym ScheduleVisualModeToolbar

Reply:
@pectom kminiliśmy to z @Qwebeck i przyszedliśmy do tego że jak ten komponent jest parent'om dla tabów, które zawierają w sobie managementPage, to jest to nieintuicyjne że zawiera w swojej nazwie Schedule, ale nic lepszego od view-only nie zostało wymyślone :D

Reply:
@pectom więc pytanie do nazewnictwa raczej pozostaje otwarte, ale nie ruszałbym go teraz ze względu na to że widzę na figmie nowy design, i na przykład wygłąda na to że taby będą widoczne nawet w edit-mode, co powoduję zmianę drzewa parent-child i wtedy można będzie wprowdzić odpowiednie nazwy :)

Reply:
Jasne to już kosmetyka, gdyby nie te przyciski to nawet bym o tym nie pisał. Niech zostanie tak jak jest.